docker/Dockerfile

```diff
@@ -1,11 +1,11 @@
 # This Dockerfile will install some additional packages on top of standard FEniCS dev image
-# Some of these are necessary to run stubs, others are for convenience
+# Some of these are necessary to run smart, others are for convenience
 # Some settings (bash aliases and vim) are also set here
 #
-# To build the image, run in the main stubs directory:
-# $ docker build -t stubs:latest -f Dockerfile .
+# To build the image, run in the main smart directory:
+# $ docker build -t smart:latest -f Dockerfile .
 # To run the image (with default settings, e.g. mounting your home directory):
-# $ docker run -ti --name stubs --init -p 8080:8888 -v $HOME:/home/fenics/shared stubs:latest
+# $ docker run -ti --name smart --init -p 8080:8888 -v $HOME:/home/fenics/shared smart:latest
 
 # Base fenics dev image
 FROM quay.io/fenicsproject/dev
```

Comment:
General comment, should we update this to use the scientific-computing images?

Reply:
Yes, I have been using updated docker files with the scientific computing images, so probably worth specifying that here. I recommend we just have a single cleaned up docker file in this repo to have less confusion, I'll add some suggested edits accordingly.

Reply:
Or we could build a new docker image width `smart` similar to https://github.com/ComputationalPhysiology/simcardems/blob/main/.github/workflows/docker-image.yml ?

Reply:
I think your suggestion (@finsberg) is sensible. We should build those at release, and publish them to github packages as usual.

Reply:
Resolved by [bd19ac0](https://github.com/RangamaniLabUCSD/stubs/pull/84/commits/bd19ac057d60a132e9caafe43a234a252ed4593b)